src/Umbraco.Web.UI.Client/src/packages/block/block-grid/components/block-grid-entries/block-grid-entries.element.ts

```diff
@@ -152,6 +145,30 @@ export class UmbBlockGridEntriesElement extends UmbFormControlMixin(UmbLitElemen
 	#rangeUnderflowValidator?: UmbFormControlValidatorConfig;
 	#rangeOverflowValidator?: UmbFormControlValidatorConfig;
 
+	@state()
```

Comment:
Nice with some clean up/formatting. But I would normally have the properties at the top and the states below? So I guess to not just make this indifferent, then maybe it would make sense to review if we in general want to switch that around?

Reply:
I'll revert it back. 👍